tests/perfsys.ts

```diff
@@ -20,29 +20,9 @@ module perftest {
     export var getCurrentDirectory = ts.sys.getCurrentDirectory;
     var exit = ts.sys.exit;
 
-    var args = ts.sys.args;

```

Comment:
There was a lot of hackery just to parse:
node .\built\local\perftsc.js --logio info.json b.ts
Then
node .\built\local\perftsc.js info.json

The args had to be 'reset' to be passed back to the compiler. As I was trying to fix it, didn't understand what it was trying to do. My understanding is you can benchmark the raw compile time, excluding any IO time spent fetching a file from the system.